pavelib/pep8.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+
+from pavelib.utils.envs import Env
+from pavelib.pylint import get_clean_report_directory
+from pavelib.utils.cmd import shell
+
+
+def pep8(services=None):
+    services = services or []
+    services = ' '.join(services)
+    Env.METRICS_DIR.makedirs_p()
+    report_dir = get_clean_report_directory('pep8')
+    report_file = "{report_dir}/pep8.report".format(
+        report_dir=report_dir,
+    )
+    shell(
+        'pep8',
+        services,
+        '|',
+        'tee',
```

Comment:
Per above, we're not appending since PEP8 is only called once per
`run_pep8` invocation. We also no longer explicitly truncate the file
before writing.